Simplified/StringUtilities.h

```diff
@@ -0,0 +1,15 @@
+//
+//  StringUtilities.h
+//  Simplified
+//
+//  Created by Aferdita Muriqi on 6/20/17.
+//  Copyright © 2017 NYPL Labs. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
For imports, we should be using the newer style nowadays as we are elsewhere in the project:

```objective-c
@import Foundation;
````